symphony/content/content.systemauthors.php

```diff
@@ -340,9 +340,12 @@ public function __form()
 
             $options = array(
                 array('author', false, __('Author')),
-                array('manager', $author->isManager(), __('Manager'))
             );
 
+            if ($isOwner || Symphony::Author()->isDeveloper() || $author->isManager()) {
```

Comment:
I think it should be 

```php
if (Symphony::Author()->isDeveloper() || ($isOwner && $author->isManager())) 
```

Reply:
Removing `$isOwner` will set the only choice in the form to `Author`. This is a problem if a user is a Manager and he want to edit his own profile. This condition will force the HTML form to be set at `author` and will set is account accordingly.

What we could do tho, is removing the select box to everyone except the developers and force the `user_type` accordingly of the context (editing his own account, or someone else)

Reply:
With my rewrite, it will force in the html only authors when the manager edits other account then itself.